docs/content.zh/docs/dev/table/sql/queries/window-deduplication.md

```diff
@@ -101,15 +102,16 @@ Flink SQL> SELECT *
 +------------------+-------+------+-------------+------------------+------------------+--------+
 ```
 
-*Note: in order to better understand the behavior of windowing, we simplify the displaying of timestamp values to not show the trailing zeros, e.g. `2020-04-15 08:05` should be displayed as `2020-04-15 08:05:00.000` in Flink SQL Client if the type is `TIMESTAMP(3)`.*
+*注意： 为了更好地理解窗口行为，这里把timestamp值后面的0去掉了。例如：在Flink SQL Client中，如果类型是 `TIMESTAMP(3)` ，`2020-04-15 08:05` 应该显示成 `2020-04-15 08:05:00.000`。*
+
+## 限制
 
-## Limitation
+## 在窗口表值函数后直接进行窗口去重的限制
 
-### Limitation on Window Deduplication which follows after Windowing TVFs directly
-Currently, if Window Deduplication follows after [Windowing TVF]({{< ref "docs/dev/table/sql/queries/window-tvf" >}}), the [Windowing TVF]({{< ref "docs/dev/table/sql/queries/window-tvf" >}}) has to be with Tumble Windows, Hop Windows or Cumulate Windows instead of Session windows. Session windows will be supported in the near future.
+目前，Flink只支持在滚动，滑动和累计[窗口表值函数]({{< ref "docs/dev/table/sql/queries/window-tvf" >}})后进行窗口去重。会话窗口不久之后就会支持。
 
-### Limitation on time attribute of order key
-Currently, Window Deduplication requires order key must be [event time attribute]({{< ref "docs/dev/table/concepts/time_attributes" >}}#event-time) instead of [processing time attribute]({{< ref "docs/dev/table/concepts/time_attributes" >}}#processing-time). Ordering by processing-time would be supported in the near future.
+### 根据时间属性排序的限制
 
+目前，Flink只支持根据[事件时间属性]({{< ref "docs/dev/table/concepts/time_attributes" >}}#event-time)排序。根据处理时间排序不久之后就会支持。
```

Comment:
It is recommended to add a space between English words and Chinese, except for Chinese punctuation. This will bring a better experience in most display modes.

For example, add a space between "Flink" and "只", and add a space before "[" and after ")",

```
目前，Flink 只支持根据 [事件时间属性]({{< ref "docs/dev/table/concepts/time_attributes" >}}#event-time) 排序。根据处理时间排序不久之后就会支持。
```

Reply:
Thanks for your suggestions and I've followed. 
I will modify all previous pull requests that can be improved according to your suggestions.